src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -79,6 +79,16 @@ public function __construct(bool $magicCall = false, bool $throwExceptionOnInval
         $this->ignoreInvalidProperty = !$throwExceptionOnInvalidPropertyPath;
     }
 
+    public function getValues($objects, $propertyPath)
```

Comment:
You should add a `iterable` typehint here and a iterable return hint and works with generators here (yield with iterating over `$objects`.

Reply:
Moreover, to stay consistent with the component's method, it should be `$objectsOrArrays` or something like that